jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/DefaultServlet.java

```diff
@@ -1077,6 +1084,23 @@ private HttpServletResponse getServletResponse(Response response)
         }
     }
 
+    private static String getIncludedPathInContext(HttpServletRequest request, String includedServletPath, boolean isPathInfoOnly)
+    {
+        String servletPath = isPathInfoOnly ? "/" : includedServletPath;
+        String pathInfo = (String)request.getAttribute(RequestDispatcher.INCLUDE_PATH_INFO);
+        if (servletPath == null)
```

Comment:
What is the case when `servletPath` is null?  If `isPathInfoOnly` is true, then the it is "/". If iPIO is false, then it is `includedServletPath`, which is never null because it is tested before calling this method.

Reply:
yep I don't think the `if` block is needed anymore